### ISSUE TITLE
New deployment stuck on MDS 'creating'

### ISSUE THREAD

**Issue by jeevadotnet** [ceph/ceph-ansible#6001]
**Bug Report**
**What happened:**
Building ceph with ceph-ansible 5.0 stable (2020/11/03) and (2020/10/28)
Once the deployment is done the MDS status is stuck in "creating". 
A 'crashed' container also appears

```ubuntu@A-08-34-cephctl:~$ sudo docker ps
CONTAINER ID        IMAGE                        COMMAND                  CREATED             STATUS              PORTS               NAMES
c642b6845126        ceph/daemon:latest-octopus   "/usr/bin/ceph-crash"    5 minutes ago       Up 5 minutes                            ceph-crash-A-08-34-cephctl
3a4570e11ca8        ceph/daemon:latest-octopus   "/opt/ceph-container…"   6 minutes ago       Up 6 minutes                            ceph-mds-A-08-34-cephctl
8e968bf608cd        ceph/daemon:latest-octopus   "/opt/ceph-container…"   13 minutes ago      Up 12 minutes                           ceph-mgr-A-08-34-cephctl
d7c805be361a        ceph/daemon:latest-octopus   "/opt/ceph-container…"   14 minutes ago      Up 13 minutes                           ceph-mon-A-08-34-cephctl
```
`sudo docker images`
```
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
ceph/daemon         latest-octopus      9dc2ed8ddd89        15 hours ago        1.06GB
```

`sudo docker inspect c642b6845126`
```json
[
    {
        "Id": "c642b6845126f0393e148b98d3c27cdf09bebd65069005513bf403a4076553ea",
        "Created": "2020-11-03T10:47:41.465898853Z",
        "Path": "/usr/bin/ceph-crash",
        "Args": [],
        "State": {
            "Status": "running",
            "Running": true,
            "Paused": false,
            "Restarting": false,
            "OOMKilled": false,
            "Dead": false,
            "Pid": 16251,
            "ExitCode": 0,
            "Error": "",
            "StartedAt": "2020-11-03T10:47:41.609613769Z",
            "FinishedAt": "0001-01-01T00:00:00Z"
        },
        "Image": "sha256:9dc2ed8ddd89d4434742edd9ca060237fa47d0a1a63c9758d0e3ae0bbfecb86a",
        "ResolvConfPath": "/var/lib/docker/containers/c642b6845126f0393e148b98d3c27cdf09bebd65069005513bf403a4076553ea/resolv.conf",
        "HostnamePath": "/var/lib/docker/containers/c642b6845126f0393e148b98d3c27cdf09bebd65069005513bf403a4076553ea/hostname",
        "HostsPath": "/var/lib/docker/containers/c642b6845126f0393e148b98d3c27cdf09bebd65069005513bf403a4076553ea/hosts",
        "LogPath": "/var/lib/docker/containers/c642b6845126f0393e148b98d3c27cdf09bebd65069005513bf403a4076553ea/c642b6845126f0393e148b98d3c27cdf09bebd65069005513bf403a4076553ea-json.log",
        "Name": "/ceph-crash-A-08-34-cephctl",
        "RestartCount": 0,
        "Driver": "overlay2",
        "Platform": "linux",
        "MountLabel": "",
        "ProcessLabel": "",
        "AppArmorProfile": "docker-default",
        "ExecIDs": null,
        "HostConfig": {
            "Binds": [
                "/var/lib/ceph:/var/lib/ceph:z",
                "/etc/localtime:/etc/localtime:ro"
            ],
            "ContainerIDFile": "",
            "LogConfig": {
                "Type": "json-file",
                "Config": {}
            },
            "NetworkMode": "host",
            "PortBindings": {},
            "RestartPolicy": {
                "Name": "no",
                "MaximumRetryCount": 0
            },
            "AutoRemove": true,
            "VolumeDriver": "",
            "VolumesFrom": null,
            "CapAdd": null,
            "CapDrop": null,
            "Capabilities": null,
            "Dns": [],
            "DnsOptions": [],
            "DnsSearch": [],
            "ExtraHosts": null,
            "GroupAdd": null,
            "IpcMode": "private",
            "Cgroup": "",
            "Links": null,
            "OomScoreAdj": 0,
            "PidMode": "",
            "Privileged": false,
            "PublishAllPorts": false,
            "ReadonlyRootfs": false,
            "SecurityOpt": null,
            "UTSMode": "",
            "UsernsMode": "",
            "ShmSize": 67108864,
            "Runtime": "runc",
            "ConsoleSize": [
                0,
                0
            ],
            "Isolation": "",
            "CpuShares": 0,
            "Memory": 0,
            "NanoCpus": 0,
            "CgroupParent": "",
            "BlkioWeight": 0,
            "BlkioWeightDevice": [],
            "BlkioDeviceReadBps": null,
            "BlkioDeviceWriteBps": null,
            "BlkioDeviceReadIOps": null,
            "BlkioDeviceWriteIOps": null,
            "CpuPeriod": 0,
            "CpuQuota": 0,
            "CpuRealtimePeriod": 0,
            "CpuRealtimeRuntime": 0,
            "CpusetCpus": "",
            "CpusetMems": "",
            "Devices": [],
            "DeviceCgroupRules": null,
            "DeviceRequests": null,
            "KernelMemory": 0,
            "KernelMemoryTCP": 0,
            "MemoryReservation": 0,
            "MemorySwap": 0,
            "MemorySwappiness": null,
            "OomKillDisable": false,
            "PidsLimit": null,
            "Ulimits": null,
            "CpuCount": 0,
            "CpuPercent": 0,
            "IOMaximumIOps": 0,
            "IOMaximumBandwidth": 0,
            "MaskedPaths": [
                "/proc/asound",
                "/proc/acpi",
                "/proc/kcore",
                "/proc/keys",
                "/proc/latency_stats",
                "/proc/timer_list",
                "/proc/timer_stats",
                "/proc/sched_debug",
                "/proc/scsi",
                "/sys/firmware"
            ],
            "ReadonlyPaths": [
                "/proc/bus",
                "/proc/fs",
                "/proc/irq",
                "/proc/sys",
                "/proc/sysrq-trigger"
            ]
        },
        "GraphDriver": {
            "Data": {
                "LowerDir": "/var/lib/docker/overlay2/e167a33da8f7561164865b50151d74044c7565cf2b09880f5f59e9be5ee822d1-init/diff:/var/lib/docker/overlay2/8d38f539391ce85b77400d32e0ad71325ce033332ca72e6635eb4d0877866388/diff:/var/lib/docker/overlay2/901abba98fd5ae542d836255ffd45f9a13e4bf49325a10f67637862792f6d304/diff:/var/lib/docker/overlay2/283b6f6ef9947f7dd5eb1026c23654412a80c3cbc877eb4c88573af50150d892/diff:/var/lib/docker/overlay2/6295e11d9fe5bb6584c4040c604cd33c64431df350dcf7ca013af7b1d020f268/diff:/var/lib/docker/overlay2/9382c073b7cff1ef5558f6b927a7cc758920c3bc3e698b5b38203bb6d3a3be26/diff:/var/lib/docker/overlay2/dbdf368b4f0388f0ff1efb10fe0f8e60317faba7b572e44db98dc9390b612516/diff:/var/lib/docker/overlay2/ab8c6a53179ad3cded500d436f9ad90cb829c9df44604d530ae2b8cc877b2439/diff:/var/lib/docker/overlay2/a394c247d763fd78930ac9c1025f83f8c3bab4ca25af7f5e5b87a7d8fec63a0d/diff:/var/lib/docker/overlay2/4b225e5c188ebd31334578efe2ca80868a0739bc3032b527f5e3502014de2fd1/diff:/var/lib/docker/overlay2/87c77a491cced43960ba9be30f225238c75dae642294d8cc43ffc33daa9957a6/diff",
                "MergedDir": "/var/lib/docker/overlay2/e167a33da8f7561164865b50151d74044c7565cf2b09880f5f59e9be5ee822d1/merged",
                "UpperDir": "/var/lib/docker/overlay2/e167a33da8f7561164865b50151d74044c7565cf2b09880f5f59e9be5ee822d1/diff",
                "WorkDir": "/var/lib/docker/overlay2/e167a33da8f7561164865b50151d74044c7565cf2b09880f5f59e9be5ee822d1/work"
            },
            "Name": "overlay2"
        },
        "Mounts": [
            {
                "Type": "bind",
                "Source": "/var/lib/ceph",
                "Destination": "/var/lib/ceph",
                "Mode": "z",
                "RW": true,
                "Propagation": "rprivate"
            },
            {
                "Type": "bind",
                "Source": "/etc/localtime",
                "Destination": "/etc/localtime",
                "Mode": "ro",
                "RW": false,
                "Propagation": "rprivate"
            }
        ],
        "Config": {
            "Hostname": "A-08-34-cephctl",
            "Domainname": "",
            "User": "",
            "AttachStdin": false,
            "AttachStdout": true,
            "AttachStderr": true,
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": [
                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
                "CEPH_VERSION=octopus",
                "CEPH_POINT_RELEASE=",
                "CEPH_DEVEL=false",
                "CEPH_REF=octopus",
                "OSD_FLAVOR=default"
            ],
            "Cmd": null,
            "Image": "docker.io/ceph/daemon:latest-octopus",
            "Volumes": null,
            "WorkingDir": "/",
            "Entrypoint": [
                "/usr/bin/ceph-crash"
            ],
            "OnBuild": null,
            "Labels": {
                "CEPH_POINT_RELEASE": "",
                "GIT_BRANCH": "HEAD",
                "GIT_CLEAN": "False",
                "GIT_COMMIT": "199c1bfb40655ace3e06f9c7b719b392f0c0d728",
                "GIT_REPO": "https://github.com/ceph/ceph-container.git",
                "RELEASE": "master-199c1bf",
                "ceph": "True",
                "maintainer": "Dimitri Savineau <dsavinea@redhat.com>",
                "org.label-schema.build-date": "20200809",
                "org.label-schema.license": "GPLv2",
                "org.label-schema.name": "CentOS Base Image",
                "org.label-schema.schema-version": "1.0",
                "org.label-schema.vendor": "CentOS"
            }
        },
        "NetworkSettings": {
            "Bridge": "",
            "SandboxID": "472b80c2e5ec4fb4a8e9bcd5fe7acd7c81e2bc4750b49cb46d833c4c9ef04435",
            "HairpinMode": false,
            "LinkLocalIPv6Address": "",
            "LinkLocalIPv6PrefixLen": 0,
            "Ports": {},
            "SandboxKey": "/var/run/docker/netns/default",
            "SecondaryIPAddresses": null,
            "SecondaryIPv6Addresses": null,
            "EndpointID": "",
            "Gateway": "",
            "GlobalIPv6Address": "",
            "GlobalIPv6PrefixLen": 0,
            "IPAddress": "",
            "IPPrefixLen": 0,
            "IPv6Gateway": "",
            "MacAddress": "",
            "Networks": {
                "host": {
                    "IPAMConfig": null,
                    "Links": null,
                    "Aliases": null,
                    "NetworkID": "07ae35cae3ceb71a5c2e957ea326c84e8647b6ee11b2d39e67cadda1e97a272a",
                    "EndpointID": "c1590314ed8944558711470d2cc746ef9d04a645339d8b85b165931e7da9df20",
                    "Gateway": "",
                    "IPAddress": "",
                    "IPPrefixLen": 0,
                    "IPv6Gateway": "",
                    "GlobalIPv6Address": "",
                    "GlobalIPv6PrefixLen": 0,
                    "MacAddress": "",
                    "DriverOpts": null
                }
            }
        }
    }
]
```

**What you expected to happen:**
Ceph with cephfs in a working order

**How to reproduce it (minimal and precise):**
Latest ceph-ansible 5.0 stable (2020/11/03 or 10/28)
`ansible-playbook -v -i /opt/ceph-ansible/inventory -e 'ansible_python_interpreter=/usr/bin/python3' /opt/ceph-ansible/site-container.yml`

- Group_vars - https://pastebin.com/Wr6mL0Fh
- Inventory - https://pastebin.com/vY36uj1y
- Ansible.log (-vvv) - https://drive.google.com/file/d/1BiWHEpEmLVXGy4f4gLV1F7OurlKhCAwK/view?usp=sharing

**Environment**:
* OS: `Ubuntu 18.04.5 LTS`
* Kernel `5.4.0-48-generic #52~18.04.1-Ubuntu SMP Thu Sep 10 12:50:22 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux`
* Docker version: `latest docker that comes with ceph-ansible` 
* Ansible version: `ansible-playbook 2.9.0`
* ceph-ansible version: `stable-5.0 (2020/11/03)`
* Ceph version (e.g. `ceph -v`): `15.2.5 `

**Comment by jeevadotnet** [comment]
duplicate of #5937
